apps/etterlatte-beregning/src/test/kotlin/beregning/BeregnBarnepensjonServiceTest.kt

```diff
@@ -45,11 +45,25 @@ internal class BeregnBarnepensjonServiceTest {
     private val beregningsGrunnlagService = mockk<BeregningsGrunnlagService>()
     private val trygdetidKlient = mockk<TrygdetidKlient>()
     private val featureToggleService = mockk<FeatureToggleService>()
-    private lateinit var beregnBarnepensjonService: BeregnBarnepensjonService
 
     @BeforeEach
     fun setup() {
-        beregnBarnepensjonService = BeregnBarnepensjonService(
+        every {
```

Comment:
Her trur eg vi gjer livet enklare for oss sjølve med å bruke _DummyFeatureToggleService_, ca sånn:

private val featureToggleService = DummyFeatureToggleService()

```
@BeforeEach
...
featureToggleService.settBryter(BeregnBarnepensjonServiceFeatureToggle.BrukInstitusjonsoppholdIBeregning,
                false)
featureToggleService.settBryter(BeregnBarnepensjonServiceFeatureToggle.BrukFaktiskTrygdetid,
                false)
...
```